concepts/05 UI Components/Form/01 Getting Started with Form/70 Validate the Form Data.md

```diff
@@ -0,0 +1,172 @@
+DevExtreme includes a validation engine that checks edited values before they are saved. This engine supports different [validation rule types](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/). 
+
+To apply validation rules to a simple item, specify them in the [validationRules[]](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#validationRules) array. You can specify an item's [isRequired](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#isRequired) property to implicitly apply the [RequiredRule](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RequiredRule/) to it. An editor is validated when its value changes.
```

Comment:
The last sentence in this paragraph probably belongs somewhere in the first paragraph. Looks like a common information about validation, not applied to the RequiredRule alone.

Reply:
Good point, I'll move this sentence to the first paragraph.